src/components/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -15,11 +16,13 @@ export default {
         placeholder: 'Some placeholder',
         readonly: false,
         clear: false,
+        position: Position.FLOATING,
```

Comment:
Maybe add two stories to show the position differences of the toolbar.

Reply:
I have concerns to such test they are prone to failures which are not even test failures but you have to adjust them even if it does not show the real failure. For example, the position is gonna be fixed on some text element and if the text changes that does not mean the Plugin is broken but the test is failing. Would like to avoid such tests which are in reality not a test. How do you feel about that?

Reply:
I guess what celine meant is not to add tests for that but add stories to have a demo in storybook where the position is top or botton.
But for me it is ok like it is now. Because you can change the position in the action menu in storybook.

Reply:
Sorry misunderstood the point, Added a story for Positioning with fixed Top